k8s/eks/yaml/multus-cm-calico.yaml

```diff
@@ -0,0 +1,54 @@
+kind: ConfigMap
```

Comment:
I think this needs to be edited further as we were facing some issues with calico scaling. Feel free to skip for now, and we can also update the `.env` file to use `AWS VPC CNI / weave` combo as the default one until this is resolved.

Reply:
Why do we even talk about calico anymore? I thought we agreed on using AWS VPC CNI + weave.

If dropping the idea that we could use either AWS VPC CNI or calico allows us to simplify the code, please do it.

Reply:
Removed everything related to calico from all scripts and yaml files.